gradle.properties

```diff
@@ -0,0 +1,7 @@
+description='Kafka UI'
+version=0.1.0-SNAPSHOT
```

Comment:
```suggestion
version=0.0.1-SNAPSHOT
```
to match the one in maven's pom

Reply:
0.1.0 is more correct version in this case

Reply:
why? in our poms it's 0.0.1